Code/Tools/ProjectManager/Source/GemCatalog/GemItemDelegate.cpp

```diff
@@ -450,7 +489,14 @@ namespace O3DE::ProjectManager
         else
         {
             painter->setFont(standardFont);
-            painter->setPen(m_textColor);
+            if(greyedOut)
+            {
+                painter->setPen(m_greyedOutTextColor);
+            }
+            else
+            {
+                painter->setPen(m_textColor);
+            }
```

Comment:
This is a bit tricky, once we go down the road of multiple colors we probably don't want to have that logic in this helper function.  Right now it's just the 2 colors, but worth thinking ahead - if we have a warning color or an error color or hyperlink color etc we probably don't want to have a large `if` block in here

Reply:
this might tie into your suggestion earlier with using color as an explicit parameter. Doing that may require adding that as a parameter at every other call site (which is technically outside scope of PR), but that may help with the long run.

Reply:
actually... I removed the only portion of text that would require a greyed out color. So I can just remove the greyed out logic altogether.